### PR TITLE
Implement basic map

### PR DESCRIPTION
**Pull request by jere1862**
#3

### CODE REVIEW

File: src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*{
+    font-family: 'Roboto';
+}
```
Comment (alarochelle) [inline src/app/app.component.css]: Newline at EOF and should be in styles.css

File: src/app/app.component.html [hunk 2]
```diff
@@ -1 +1 @@
-<router-outlet></router-outlet>
+<router-outlet></router-outlet>
```
Comment (alarochelle) [inline src/app/app.component.html]: Newline at EOF

File: src/app/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -5,3 +5,5 @@
     <md-icon>language</md-icon>
   </button>
 </md-toolbar>
+
+<app-map></app-map>
```
Comment (alarochelle) [inline src/app/dashboard/dashboard.component.html]: Newline at EOF

File: src/app/app.component.ts [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
-
 import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: [
+    './app.component.css',
+    '../../node_modules/roboto-fontface/css/roboto/roboto-fontface.css',
```
Comment (alarochelle) [inline src/app/app.component.ts]: styles.css

File: src/app/app.module.ts [hunk 5]
```diff
@@ -32,6 +37,6 @@ export function HttpLoaderFactory(http: HttpClient): TranslateHttpLoader {
     })
   ],
   providers: [],
-  bootstrap: [AppComponent]
+  bootstrap: [AppComponent],
```
Comment (alarochelle) [inline src/app/app.module.ts]: no need for virgule

File: src/app/dashboard/dashboard.component.spec.ts [hunk 6]
```diff
@@ -1,14 +1,54 @@
...
+}
 
 describe('DashboardComponent', () => {
   let component: DashboardComponent;
   let fixture: ComponentFixture<DashboardComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ DashboardComponent ]
+      declarations: [ DashboardComponent, MapComponent, MockPipe],
```
Comment (alarochelle) [inline src/app/dashboard/dashboard.component.spec.ts]: manque un espace avant ton ]

File: src/app/map/map.component.css [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  height: 50vh;
+}
+
+.map-holder{
+  width: 80%;
+  margin-top: 1%;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+}
```
Comment (alarochelle) [inline src/app/map/map.component.css]: Newline at EOF

File: src/app/map/map.component.html [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<md-card class="map-holder mat-elevation-z6">
+  <md-card-content>
+    <agm-map [latitude]="lat" [longitude]="lng">
+      <agm-marker [latitude]="lat" [longitude]="lng"></agm-marker>
+    </agm-map>
+  </md-card-content>    
+</md-card>
```
Comment (alarochelle) [inline src/app/map/map.component.html]: Newline at EOF

File: src/app/map/map.component.css [hunk 9]
```diff
@@ -0,0 +1,11 @@
+agm-map {
+  height: 50vh;
+}
+
+.map-holder{
+  width: 80%;
+  margin-top: 1%;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (alarochelle) [inline src/app/map/map.component.css]: privilégie `margin: 1% auto 0`

File: README.md [hunk 10]
```diff
@@ -1,7 +1,4 @@
 # MixingConsoleFrontend
-
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.4.2.
```
Comment (alarochelle) [inline README.md]: tu as osé

File: src/app/dashboard/dashboard.component.spec.ts [hunk 11]
```diff
@@ -1,14 +1,54 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { DashboardComponent } from './dashboard.component';
+import { Pipe, PipeTransform } from '@angular/core';
+import { MaterialModule } from '../material.module';
```
Comment (alarochelle) [inline src/app/dashboard/dashboard.component.spec.ts]: Unused, non?

Comment (jere1862) [reply src/app/dashboard/dashboard.component.spec.ts]: removed it

**Pull request merged**